src/helpers/rc.ts

```diff
@@ -0,0 +1,79 @@
+import { Client } from './../client'
+import { Account } from './../steem/account'
+import { RCAccount, RCParams, RCPool } from './../steem/rc'
+import { getVests } from '../steem/misc'
+
+export class RCAPI {
+    constructor(readonly client: Client) { }
+    /**
+   * Convenience for calling `rc_api`.
+   */
+    public call(method: string, params?: any) {
+        return this.client.call('rc_api', method, params)
+    }
+    /**
+    * Returns RC data for array of usernames
+    */
+    public async findRCAccounts(usernames: string[]): Promise<Array<RCAccount>> {
+        let x = await this.call('find_rc_accounts', { accounts: usernames })
+        return x['rc_accounts'] ? x['rc_accounts'] as Array<RCAccount> : x
```

Comment:
Why the ternary if statement here?

Reply:
This call returns { rc_accounts }. I'm checking if the object has rc_accounts, otherwise, it's an error and the error will be returned.

Reply:
Errors should always be thrown never returned. But I'm wondering if you actually can get anything except an rc_accounts list there. Errors should be handled already at the RPC level

Reply:
Oh ok, good to know. Will remove the ifs